### PR TITLE
Fix defunct download server link

### LINKED ISSUE #158: OpenCPN download functionality does not work
Tools->Download->OpenCPN (Kap) points to the defunct server https://downloadlayer--onlinechart-ol3.netlify.app/ .

A simple workaround would be to instead point to [alpha](http://alpha.openseamap.org/#/download/.*/2040575595?lat=27.1957&lon=-12.0683&zoom=3&layers=A00000011000&lang=en).

### PR DESCRIPTION
**Pull request by wschildbach**
Replace link with one to alpha.openseamap.org. This fixes #158 .

### CODE REVIEW

**Review by stevo01 — COMMENTED** [review]
I think that a good workaround after "downloadlayer--onlinechart-ol3.netlify.com" is not available any more.

**Comment by wschildbach** [comment]
@aAXEe would you like to merge this one?

File: classes/topmenu.inc [hunk 1]
```diff
@@ -83,7 +83,7 @@
                   <a href="http://openseamap.org/index.php?id=kartendownload"><strong>Download</strong></a>
                 </li>
             <li>
-              <a href="https://downloadlayer--onlinechart-ol3.netlify.com" target="_blank">OpenCPN (Kap)</a>
+              <a href="https://alpha.openseamap.org/#/download/.*/2040575595?lat=27.1957&lon=-12.0683&zoom=3&layers=A00000011000&lang=en" target="_blank">OpenCPN (Kap)</a>
```
Comment (aAXEe) [inline classes/topmenu.inc:86]:
```suggestion
              <a href="https://alpha.openseamap.org/#/download/.*/null?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en" target="_blank">OpenCPN (Kap)</a>
```

One map was selected with the other link.

**Review by aAXEe — CHANGES_REQUESTED** [review]

File: classes/topmenu.inc (hunk 1 above)
Comment (wschidol) [reply classes/topmenu.inc:86]: That makes sense, thank you. When I use

> https://alpha.openseamap.org/#/download/.*/null?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en

the server redirects to 

> https://alpha.openseamap.org/#/download/.*/NaN?lat=35.0245&lon=-63.2636&zoom=3&layers=A10001011000&lang=en

So not sure whether null or NaN is preferred but it works so agreed.

**Review by aAXEe — APPROVED** [review]

**Pull request merged**